### PR TITLE
sscs 8622 New config for collection of dwp docs

### PR DESCRIPTION
**Pull request by chrisdavecm**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-8622

### Change description ###
Changed config to match new model for dwp documents

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml [hunk 1]
```diff
@@ -10,13 +10,20 @@ sort:
 hasFolderCoversheets: false
 pageNumberFormat: numberOfPages
 documentNameValue: /documentFileName
+documentLinkValue: /editedDocumentLink
```
Comment (jackmaloney) [inline src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml:13]: What does this do?

File: src/main/resources/bundleconfiguration/sscs-bundle-welsh-config.yaml [hunk 2]
```diff
@@ -13,10 +13,16 @@ documentNameValue: /documentFileName
...
-      - type: document
-        property: /dwpResponseDocument
-      - type: document
-        property: /dwpEvidenceBundleDocument
+      - type: documentSet
+        property: /dwpDocuments
+        filters:
+          - property: /documentType
+            value: dwpEvidenceBundle
+      - type: documentSet
```
Comment (jackmaloney) [inline src/main/resources/bundleconfiguration/sscs-bundle-welsh-config.yaml:21]: You've switched the order

**Review by jackmaloney — CHANGES_REQUESTED** [review]
You've switched the order - should be response first followed by evidence bundle

File: src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml (hunk 1 above)
Comment (chrisdavecm) [reply src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml:13]: This is where we got lucky, by default it looks for the document url at property documentLink, this allows us to look at the property editedDocumentLink for the link for the edited document. If it didn't do this the model change wouldn't have worked.

**Review by yogesh-hullatti — APPROVED** [review]

File: src/main/resources/bundleconfiguration/sscs-new-bundle-config.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+pageNumberFormat: numberOfPages
+documentNameValue: /documentFileName
+folders:
+  - name: DWP
+    documents:
+      - type: documentSet
+        property: /dwpDocuments
+        filters:
+          - property: /documentType
+            value: dwpEvidenceBundle
```
Comment (jackmaloney) [inline src/main/resources/bundleconfiguration/sscs-new-bundle-config.yaml:20]: Still wrong order

File: src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml (hunk 1 above)
Comment (jackmaloney) [reply src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml]: Ha ok

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**